DAL/Repositories/Implementations/ModelNameRepository.cs

```diff
@@ -19,24 +19,31 @@ public async Task<IEnumerable<ModelName>> GetRangeAsync(int page, int pageSize,
             .AsNoTracking()
             .ToListAsync(cancellationToken);
     }
-
+    
```

Comment:
Not a big issue, just try to keep code clean.
Also setup the "Cleanup code" setting in VS